src/main/kotlin/org/rust/ide/colors/RustColors.kt

```diff
@@ -9,11 +9,11 @@ import com.intellij.openapi.editor.DefaultLanguageHighlighterColors as Default
  */
 enum class RustColor(humanName: String, val default: TextAttributesKey) {
     IDENTIFIER            ("Identifier",                  Default.IDENTIFIER),
-    FUNCTION_DECLARATION  ("Function declaration",        Default.FUNCTION_DECLARATION),
-    INSTANCE_METHOD       ("Instance method declaration", Default.INSTANCE_METHOD),
-    STATIC_METHOD         ("Static method declaration",   Default.STATIC_METHOD),
-    PARAMETER             ("Function parameter",          Default.PARAMETER),
-    SELF_PARAMETER        ("Self parameter",              Default.KEYWORD),
+    FUNCTION_DECLARATION  ("Function",                    Default.FUNCTION_DECLARATION),
+    INSTANCE_METHOD       ("Method",                      Default.INSTANCE_METHOD),
+    STATIC_METHOD         ("Associated function",         Default.STATIC_METHOD),
+    PARAMETER             ("Function argument",           Default.PARAMETER),
```

Comment:
>Hm, interesting, I never thought of arguments/parameters in this way.

Yep, this terminological distinction really matters only when you "implement" a language, not when you use it. But it is a pretty well established terminology: 

https://en.wikipedia.org/wiki/Parameter_(computer_programming)#Parameters_and_arguments

It's definitely helps to use the precise term in the code (there are `RustParametersElement` and `RustArgListElement`). 


It's not that important for user-facing names, but I'd still prefer parameter, because Kotlin and Java use this name in color settings as well.